readiness/controller/pkg/readiness/readiness_controller.go

```diff
@@ -34,14 +36,142 @@ type ReadinessReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.6.4/pkg/reconcile
 func (r *ReadinessReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = context.Background()
-	_ = r.Log.WithValues("readiness", req.NamespacedName)
+	readiness := &corev1alpha2.Readiness{}
+	err := r.Client.Get(ctx, req.NamespacedName, readiness)
+	if err != nil {
+		r.Log.Error(err, "Error while fetching readiness")
+		return ctrl.Result{}, err
+	}
 
-	// TODO(user): your logic here
+	updateRequired := false
+	oldStatus := readiness.Status
```

Comment:
What's the reason behind naming this `oldStatus`? This is the object that's current.

Reply:
Changed